django/contrib/gis/db/backends/spatialite/operations.py

```diff
@@ -113,13 +113,13 @@ def spatial_version(self):
             raise ImproperlyConfigured('GeoDjango only supports SpatiaLite versions 4.0.0 and above.')
         return version
 
-    def convert_extent(self, box, srid):
+    def convert_extent(self, box):
         """
         Convert the polygon data received from SpatiaLite to min/max values.
         """
         if box is None:
             return None
-        shell = Geometry(box, srid).shell
+        shell = Geometry(box).shell
```

Comment:
@claudep 
Could you look at this? It looks like passing SRID is really useless here, since we already have coordinates in the needed SRID. Does that make sense?